scripts/RemoteTests.json

```diff
@@ -172,7 +172,7 @@
                 "Tls": ["schannel", "openssl"],
                 "Arch": ["x64", "x86", "arm", "arm64"],
                 "Exe": "secnetperf",
-                "Arguments": "-test:RPS -target:$RemoteAddress"
+                "Arguments": "-exec:lowlat -test:RPS -target:$RemoteAddress"
```

Comment:
RPS tests both throughput and latency, depending on the test. So I am unsure of what to use? Both?